### PR TITLE
ColorPicker: Refine design of color picker 

### PR DESCRIPTION
**Pull request by torkelo**
I noticed that the color picker has undergone a design change in v8.3. Think that change while good also highlighted a few remaining issues that could be further improved. 

old v6-v8 design:
Screenshot from 2021-12-17 12-15-53

v8.3 design: 
Screenshot from 2021-12-17 12-11-36

Problems: 
* Takes up a lot more space (esp horizontal) 
* Color names are not aligned with color circles 
* It has no border like other popovers (a problem with old version as well) so that blends into the background too much, hard to make out the popovers borders
* Wonder if we can integrate the primary color into the color scale instead of making it separate 

Proposed tweaked design:
* Show the primary color where it is the spectrum (middle) 
* Return tabs to font size small like in v6-v8 
* Reduce padding a bit 
* Add popover border to help make it stand out 

Screenshot from 2021-12-17 12-20-31

### CODE REVIEW

File: packages/grafana-ui/src/components/ColorPicker/NamedColorsGroup.tsx [hunk 1]
```diff
@@ -19,40 +19,25 @@ const NamedColorsGroup: FunctionComponent<NamedColorsGroupProps> = ({
...
-                    isSelected={shade.name === selectedColor}
-                    color={shade.color}
-                    onClick={() => onColorSelect(shade.name)}
-                  />
-                </div>
-              )
-          )}
-        </div>
+        {hue.shades.map((shade) => (
+          <div key={shade.name}>
```
Comment (kaydelaney) [inline packages/grafana-ui/src/components/ColorPicker/NamedColorsGroup.tsx:30]: Do we need this wrapper `div`?

**Review by kaydelaney — APPROVED** [review]
Looks good! Just left a small comment

File: packages/grafana-ui/src/components/ColorPicker/NamedColorsGroup.tsx (hunk 1 above)
Comment (torkelo) [reply packages/grafana-ui/src/components/ColorPicker/NamedColorsGroup.tsx:30]: yea, this looks superfluous as well

**Comment by ryantxu** [comment]
While we are discussing this... would it be better to have more saturated on the left moving to ligher on the right?

**Comment by torkelo** [comment]
@ryantxu good point, reversed the order

**Pull request merged**